.github/workflows/ci-msix.yml

```
@@ -0,0 +1,57 @@
+  pull_request:
+    branches: [ main, master ]
+
+jobs:
+  build-msix:
+    runs-on: windows-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v3
```

actionlint (1.7.12)

[error] the runner of "actions/setup-dotnet@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)